packages/@tailwindcss-upgrade/src/index.ts

```diff
@@ -163,12 +166,13 @@ async function run() {
         await Promise.allSettled(
           files.map((file) => migrateTemplate(config.designSystem, config.userConfig, file)),
         )
-      }
 
-      success('Template migration complete.')
+        success('↳ Template migration complete.')
+      }
     }
 
     // Migrate each CSS file
+    info('Migrating stylesheets')
```

Comment:
We didn't have a start message for the stylesheet migrations, but we do for others, so I made it more complete by adding this.

This is just because I was looking at the output and noticed it.